explorer/src/components/TokenStatsCard.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { LoadingCard } from "./common/LoadingCard";
+import { ErrorCard } from "./common/ErrorCard";
+import {
+  useCoinGeckoCategoryStats,
+  COIN_GECKO_SOLANA_CATEGORY,
+} from "utils/coingecko";
+
+import { formatDollarValue } from "utils";
+import { displayTimestampWithoutDate } from "utils/date";
+
+export function TokenStatsCard() {
+  const [error, loading, categoryStats] = useCoinGeckoCategoryStats(
+    COIN_GECKO_SOLANA_CATEGORY
+  );
+
+  if (error) return <ErrorCard text={error.toString()} />;
+
+  if (loading) return <LoadingCard />;
+
+  if (categoryStats)
+    return (
+      <div className="card staking-card">
+        <div className="card-body">
+          <div className="d-flex flex-md-row flex-column">
+            <div className="p-2 flex-fill">
+              <h4>Market Capitalization</h4>
+              <h1>
+                <em>{formatDollarValue(categoryStats.market_cap)}</em>
```

Comment:
I considered using `abbreviatedNumber` here but that felt too lossy and I prefer the complete value 

Which one do you'll prefer? 

![localhost_3000_tokens_](https://user-images.githubusercontent.com/7701981/120514785-48bcdd80-c3eb-11eb-94aa-1b49218e8624.png)

![localhost_3000_tokens_ (1)](https://user-images.githubusercontent.com/7701981/120514794-4b1f3780-c3eb-11eb-8ee4-4035c9cd33aa.png)

Reply:
abbreviatedNumber I think looks better